File: notes/JS Basics/fizzBuzz.js
const first = process.argv[2];
const second = process.argv[3];

for (let i = 1; i <= 100; i++) {
  if (i % first == 0) {
    if (i % second == 0) {
      console.log("fizzbuzz")
    } else {
      console.log("fizz");
    }
  } else if (i % second == 0) {
    if (i % first != 0) {
      console.log("buzz");
    }
  } else {
    console.log(i);
  }
}

File: notes/Ruby: OOP/fizzbuzz.rb
class FizzBuzz
  attr_accessor(:first, :second)
  
  def initialize(a, b)
    @first = a
    @second = b
  end

  def run
    fizzBuzzArr = Array.new

    for i in 1..100
      toPush = ""
      if i % @first == 0
        toPush += "fizz"
      end
      if i % @second == 0
        toPush += "buzz"
      end
      if i % @first != 0 && i % @second != 0
        toPush = i
      end
      fizzBuzzArr << toPush
    end

    return fizzBuzzArr
  end

end